AsyncDisplayKit/ASControlNode.mm

```diff
@@ -77,23 +77,35 @@ - (id)init
 {
   if (!(self = [super init]))
     return nil;
-
+  
   _enabled = YES;
-
+  
   // As we have no targets yet, we start off with user interaction off. When a target is added, it'll get turned back on.
   self.userInteractionEnabled = NO;
+  
   return self;
 }
 
+- (void)didLoad
+{
+#if TARGET_OS_TV
```

Comment:
Put the gating around the entire method, so we do not even define it / have the call overhead for iOS.

Also, must call super!